### ISSUE TITLE
U+211D 18.0.0 to 21.0.0 regression

### ISSUE THREAD

**Issue by pv4** [be5invis/Iosevka#1605]
The middle column is 18.0.0, the right column is 21.0.0. The issue is present in sans/slab and upright/italic variants.

![image](https://user-images.githubusercontent.com/2502094/224865816-e9f827b4-cd59-4ede-bb47-9d7bf860a3e9.png)

**Comment by be5invis** [comment]
@pv4 Just curious... are you diffing every version? I have to say that if a shape isn't become "broken" then it will be usually an intensional change.

**Comment by pv4** [comment]
@be5invis I understand you and I try my best to avoid reporting intended design decisions.

I use https://github.com/pv4/iosevka-ttf-diff to generate diffs. Though it doesn't detect new and removed glyphs (only those present in both the older and the newer versions) it generates too many false-positives so I have to look through the results by hand anyway and for each glyph diff decide whether it's an intended change or it is (probably) a bug. Moreover, during this process I sometimes discover some glyphs that haven't changed much but are buggy in both versions.

It requires some time. From January 2023 (when I started doing it) I only tested two version changes: 18.0.0 vs 16.3.6, and 21.0.0 vs 18.0.0.

If I'm unsure whether the change is intended or not I don't report it. I only report changes and design decisions that I think can be incorrect (this was the case with e.g. #1584). Sometimes it's hard to say whether the change is intended without having "the idea" in the head. Those are the issues I'd just like to bring to your attention. I report them and expect you to close them as "This is by design. Works as intended" if they are expected.
